distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Widget_Helpers.enso

```diff
@@ -42,7 +43,7 @@ make_aggregate_column_selector table display:Display=..Always =
     ## Should be a list of Text columns only
     count_not_empty = Option "..Count_Not_Empty" "..Count_Not_Empty" [column_widget]
     count_empty = Option "..Count_Empty" "..Count_Empty" [column_widget]
-    concatenate = Option "..Concatenate" "..Concatenate" [column_widget]
+    concatenate = Option "..Concatenate" "..Concatenate" [column_widget, ["separator", make_single_choice [',', ';', '|', ['{tab}', "'\t'"], ['{space}', "' '"], ['{newline}', "\n"], '_', ['Custom', "'?'"]]]]
```

Comment:
Feel like I have seen this array 3 times in this review. Could we define it once? Maybe in this file?